Source/iOS/Extensions/UIView+Frame.swift

```diff
@@ -0,0 +1,41 @@
+import UIKit
+
+public extension UIView {
+    
+    public var height: CGFloat {
```

Comment:
May be. I was just going through the code of Example `Apple News` in `Spots`. Just saw this 

```swift
    featuredImage.frame.size.width = contentView.frame.width - 30
    featuredImage.frame.size.height = 200
    featuredImage.frame.origin.y = 15
    featuredImage.frame.origin.x = 15

    titleLabel.text = item.title
    titleLabel.frame.size.width = contentView.frame.width - 30
    titleLabel.sizeToFit()
    titleLabel.frame.origin.x = 15
    titleLabel.frame.origin.y = featuredImage.frame.maxY + 15

    introLabel.text = item.subtitle
    introLabel.frame.size.width = contentView.frame.width - 30
    introLabel.sizeToFit()
    introLabel.frame.origin.x = 15
    introLabel.frame.origin.y = titleLabel.frame.maxY + 15
```

What do you think @onmyway133 ? Is this not fit for `Sugar` ?